modules/RTC/JitsiLocalTrack.js

```diff
@@ -525,7 +525,8 @@ export default class JitsiLocalTrack extends JitsiTrack {
             }
 
             promise.then(streamsInfo => {
-                const mediaType = this.getType();
+                // The track kind for presenter track is video as well.
+                const mediaType = this.getType() === MediaType.PRESENTER ? MediaType.VIDEO : this.getType();
```

Comment:
I can't find `MediaType.PRESENTER` defined anywhere. Also I just noticed that in jitsi-meet the code overrides the track type: https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/tracks/functions.js#L49 This type should live inside LJM. `track.type` is not a public var to be modified from jitsi-meet

Reply:
Oops, I forgot to commit the changes to service/RTC/MediaType.js.